Gemfile

```diff
@@ -8,6 +8,6 @@ gem "minitest"
 gem "rspec"
 gem "mocha"
 gem "yard"
-gem "rubocop", [">= 1.0", "< 1.30"] # TODO: Our cops are broken by rubocop 1.30, we need to figure out why
+gem "rubocop", ">= 1.0"
 gem "rubocop-shopify", require: false
```

Comment:
The version of rubocop-shopify compatible with rubocop >= 1.30 requires ruby 2.7

Reply:
Yeah, rubocop-shopify is annoying for open source gems where you generally want to support some older rubies.